ja_JP/docs/recipes/when-to-use-plan.md

```diff
@@ -0,0 +1,139 @@
+___
+**訳注**
+
+これは[endpoint-testing.md](https://github.com/sindresorhus/ava/blob/master/docs/recipes/when-to-use-plan.md)の日本語訳です。こちらがAVAのmasterブランチとの差分の[リンク](https://github.com/sindresorhus/ava/compare/93af8d8d2cb48fe0d2c4ede3c92964a295f60cb6...master#diff-0c25d982e94d600cb6b8e438a0e67169)になります(このリンクをクリックして、`endpoint-testing.md`に変更点が見当たらなければ、この翻訳が最新であることを意味します)。
+___
+
+# `t.plan()`はいつ使うのか
+
+AVAと[`tap`](https://github.com/tapjs/node-tap)/[`tape`](https://github.com/substack/tape)の主な差は`t.plan()`の振る舞いです。AVAで`t.plan()`は何回アサーションが行われたのかを数えるときのみ利用されます。これによるテストの自動終了はしていません。
+
+## `t.plan()`の良くない使い方
+
+`tap`/`tape`になれた多くのユーザは`t.plan()`をすべてのテストごとに使う傾向があります。しかし、AVAではこれを「最先のやり方」だと考えていません。その代わりに、`t.plan()`がある意味を表す状況のみに使われるべきだと思います。
```

Comment:
`「最先のやり方」` should be replaced with `「ベストプラクティス」`.

```md
`t.plan()`がある意味を表す状況のみに使われるべきだと思います。
```
This would be better like below.
```md
`t.plan()`が何かしらの値を意味するときだけに使われるべきと信じています。
```